### ISSUE TITLE
SHA-1 format?

### ISSUE THREAD

**Issue by nopoz** [dmstocking/couch-tatertot#16]
What is the expected format for the SHA-1 fingerprint? 

I've tried: 

All uppercase with spaces, without spaces, with colons.

All lowercase with spaces, without spaces, with colons.

None of which work, I know it's the right SHA-1. The only way I can get HTTPS to work is to accept all certificates.

**Comment by dmstocking** [comment]
Um that is actually a good question. I didn't write most of that stuff so i will have to look. If i had to guess I would say its upper and lower with spaces and colons. Let me check who commited that and see if I cant ask him.

**Comment by nopoz** [comment]
Not sure if this will help but I'm also using transdroid (https://github.com/erickok/transdroid) which has a config section that looks almost identical - where you enter in the SHA-1. I'm using the same certificate and the accepted SHA-1 format is all uppercase with colons as separators. For example, `#A:A#:A#:A#...` etc
But this does not seem to work in CouchTaterTot.

**Comment by nopoz** [comment]
ah okay I got it working - the hostname that the certificate was signed for was not the same hostname that I had specified in the settings to connect to, once I had changed the hostname to match it connects. Both hostnames point to the same server. So it would appear the certificate validation is more strict than transdroid. The accepted format was the same as transdroid - all uppercase alpha with colons as seperators. 

Thanks so much for this great app, keep up the great work! :)
